link/uprobe.go

```diff
@@ -198,8 +194,20 @@ func (ex *Executable) address(symbol string, opts *UprobeOptions) (uint64, error
 		return 0, fmt.Errorf("cannot resolve %s library call '%s': %w "+
 			"(consider providing UprobeOptions.Address)", ex.path, symbol, ErrNotSupported)
 	}
+	return address, nil
+}
 
-	return address + opts.Offset, nil
+// addressMulti calculates the array of addresses from symbols array in the executable.
+func (ex *Executable) addressMulti(symbols []string) ([]uint64, error) {
```

Comment:
nit: rename to `uprobeMultiAddresses`, for consistency?

Reply:
hm addressMulti is not uprobe specific, it just returns offsets for symbols array,
there's:
```
  func (ex *Executable) address(symbol string) (uint64, error) {
  func (ex *Executable) addressMulti(symbols []string) ([]uint64, error) {
```

which are not probe sprecific and:

```
func (ex *Executable) uprobeAddress(symbol string, opts *UprobeOptions) (uint64, error) {
```
which returns address + opts.Offset which is specific for uprobe

uprobe multi does not provide any symbol + relative offset functionality,
it's either symbol or offset

Reply:
hmm ok, I had the impression that the only non-specific common variant was the `address(...)` one 👍🏻